src/analysis/specific_limits.lean

```diff
@@ -545,6 +545,65 @@ end
 
 end summable_le_geometric
 
+section normed_ring_geometric
+variables [normed_ring α]
+
+open normed_space
+
+/- In a normed ring, the powers of an element x with `∥x∥ < 1` tend to zero.
+Proved above (same name, different namespace) for normed fields. -/
+lemma normed_ring.tendsto_pow_at_top_nhds_0_of_norm_lt_1 (x : α) (h : ∥x∥ < 1) :
```

Comment:
There is no need to have both lemmas. Could you please replace `normed_field` version with this one? Same for the next lemma.

Reply:
This one can supersede the `normed_field` version (I will do it in my next commit).  For the next one, `summable_geometric_of_norm_lt_1`, the summability of the geometric series `∑' (i:ℕ), x ^ i` when `∥x∥ < 1`, I don't think it's possible to combine them, since neither set of hypotheses (`normed_field` versus `normed_ring` + `complete_space`) supersedes the other.  It need not hold for an incomplete normed ring; consider polynomials on [0, 1] with the sup-norm.